uwrt_mars_rover_hw_bridge/CMakeLists.txt

```diff
@@ -1,11 +1,38 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.10.2)
 project(uwrt_mars_rover_hw_bridge)
 find_package(catkin REQUIRED)
 catkin_package(
 #  INCLUDE_DIRS include
-#  LIBRARIES ar_tracker
+  LIBRARIES library/${PROJECT_NAME}/uwrt-mars-rover-hw-bridge
 #  CATKIN_DEPENDS other_catkin_pkg
 #  DEPENDS system_lib
 )
 
 add_subdirectory(library/${PROJECT_NAME})
+
+# adapted from uwrt_mars_rover_hw/CMakeLists.txt
+############################################################################################
+# Do i need this? This package cannot have an exe since it does not have a main right?
```

Comment:
this would install the static library (.a) or dynamic library (.so) that gets compiles from the ``uwrt-mars-rover-hw-bridge` target so that other cmakelists can use it

Reply:
oops i meant to put this for line 22. this install executables thing shouldnt be here cuz there's no executable